src/riak_core_coverage_plan.erl

```diff
@@ -759,6 +764,17 @@ covers(KeySpace, CoversKeys) ->
 data_bits(PartitionCount) ->
     160 - round(math:log(PartitionCount) / math:log(2)).
 
+%% @private
+
+%% Crimes against computerkind. Finds the next value of two greater
+%% than the requested value using string manipulation.
+next_power_of_two(X) ->
+    Next = 1 bsl length(hd(io_lib:format("~.2b", [X]))),
```

Comment:
log2(N) = log(N)/log(2)

````
log2(X) -> math:log(X)/math:log(2).
````
I cannot imagine that being slower than having fun with `io_lib` and `length`.